### DISCUSSION TITLE
Store active language NOT in url.

### DISCUSSION THREAD

**Discussion by anxhirr** [robisim74/qwik-speak#126] [Q&A] [answered]
Thanks @robisim74 for contributing this library to the community.

I was wondering if there is another way to store the current language rather than in url. 
To me, having the language in the url is pretty annoying.
Plus the fact that Qwik Typed Routes will create types for lang param resulting in having to pass the language everywhere.

**Comment by nelsonprsousa** [comment, marked as answer]
You can infer from cookies or accept-language header.

However you'll end up with duplicated content to search engines and get penalties for it (if SEO is important to you).

**Reply by anxhirr** [reply]
I see, thanks for the qwik reply.
